MdePkg/Include/IndustryStandard/Tpm2Acpi.h

```diff
@@ -19,46 +19,46 @@ SPDX-License-Identifier: BSD-2-Clause-Patent
 #define EFI_TPM2_ACPI_TABLE_REVISION    EFI_TPM2_ACPI_TABLE_REVISION_4
 
 typedef struct {
-  EFI_ACPI_DESCRIPTION_HEADER Header;
+  EFI_ACPI_DESCRIPTION_HEADER    Header;
   // Flags field is replaced in version 4 and above
   //    BIT0~15:  PlatformClass      This field is only valid for version 4 and above
   //    BIT16~31: Reserved
-  UINT32                      Flags;
-  UINT64                      AddressOfControlArea;
-  UINT32                      StartMethod;
+  UINT32                         Flags;
+  UINT64                         AddressOfControlArea;
+  UINT32                         StartMethod;
   UINT8                       PlatformSpecificParameters[12];
```

Comment:
@gdong1 could align space?

Reply:
Thanks for the comments. Let me update it.